pkg/encryption/path.go

```diff
@@ -58,24 +58,10 @@ func DecryptPath(path storj.Path, cipher storj.Cipher, key *storj.Key) (decrypte
 
 // DerivePathKey derives the key for the given depth from the given root key.
 // This method must be called on an unencrypted path.
-func DerivePathKey(path storj.Path, key *storj.Key, depth int) (derivedKey *storj.Key, err error) {
-	if depth < 0 {
-		return nil, Error.New("negative depth")
-	}
-
-	// do not derive key from empty path
-	if len(path) == 0 {
-		return key, nil
-	}
-
-	comps := storj.SplitPath(path)
-	if depth > len(comps) {
-		return nil, Error.New("depth greater than path length")
-	}
-
+func DerivePathKey(path storj.Path, key *storj.Key) (derivedKey *storj.Key, err error) {
```

Comment:
Could you update the godoc comment to reflect what this method now exactly does?

Reply:
Of course. I'm not sure this is staying this way, either.